src/pages/Implementation.pug

```diff
@@ -3,4 +3,50 @@ extends ../templates/contents.pug
 block headVars
     - var title = "Implementation"
     - var tagline = "A sample page for the theme."
-    - var requireMathJax = true
\ No newline at end of file
+    - var requireMathJax = true
+    
+block article
+    :markdown-it(html)
+    
+        #One of our Main goals during the iGEM project was to work as much as we could in subgroups and parallelize all of our work.
+        Thus, we created own teams for Design, Hardware, Software, Lab work, and much more, which allowed us not only to establish our lab progress but also build the hard- and software to execute it automatically.
+        In the past months, we managed to design and build a DNA synthesis system, which works with easy immobilization techniques, straightforward software, and a self-built hardware.
+        
+        We developed a synthesis method, which can be part of a complete data storage system.
+        Imagine seeing a black box, maybe the size of a regular drawer with one interface for input and one for output.
+        On the inside tons of information are stored, from the outside it is just a black box.
+        If you put in information, for example a document or an image, it would first be stored in a cache for processing purposes, then encoded into DNA-code by our software and finally be synthesized by our Hardware “DIPsy” or one of its successors.
```

Comment:
```suggestion
        If information is put in, for example, a document or an image, it would first be stored in a cache for processing purposes, then encoded into DNA-code by our software and finally be synthesized by our Hardware “DIPsy” or one of its successors.
```